projects/cds/src/merchandising/facade/cds-merchandising-product.service.spec.ts

```diff
@@ -88,13 +89,60 @@ describe('CdsMerchandisingProductService', () => {
 
   it('loadProductsForStrategy should call connector', () => {
     const strategyRequest = {
```

Comment:
like mentioned above, keep it consistent like this, where it is lowercase

Reply:
Done